starter-kit/webhook/README.md

```diff
@@ -93,7 +103,7 @@ For example, the response for `type=api` and `location=United States of America`
 }
 ```
 
-8. If you want to make a call to the Discovery service, you need to set some parameters that lets you call the IAM-enabled service. On the left, click on the **Parameters** tab. Add the following parameters: `api_key`, `url`, `collection_id`, and `env_id`. The `collection_id` and `env_id` are the values you noted from the Watson Discovery service in the previous steps.
+8. If you want to make a call to the Discovery service, you need to set some parameters that lets you call the IAM-enabled service. On the left, click on the **Parameters** tab. Add the following parameters: `api_key`, `twcApiKey`, `url`, `collection_id`, and `env_id`. The `twcApiKey` is the api key from The Weather Company and `collection_id` and `env_id` are the values you noted from the Watson Discovery service in the previous steps.
```

Comment:
1. delete the "s" at the end of "lets". 
2. Should probably make the parameters that you list out into an unordered list to make it easy to scan.
3. API key (instead of api key)